usaspending_api/references/models.py

```diff
@@ -236,8 +237,38 @@ class Location(DataSourceTrackedModel):
     def save(self, *args, **kwargs):
         self.load_country_data()
         self.load_city_county_data()
+        self.fill_missing_state_data()
         super(Location, self).save(*args, **kwargs)
 
+    def state_name_mismatch(self):
+        "True if state_name and state_code not in agreement"
+
+        if (not self.state_name) or (not self.state_code):
+            return False
+        return (self.state_name != code_to_state.get(self.state_code))
+
+    def is_us_state(self):
```

Comment:
Oh, good point.  That was required by the code to fill in missing countries, which I took out b/c I couldn't get a clear answer on whether we needed it.  I'll take it out of here.